packages/jwt-service/src/functions/index.ts

```diff
@@ -411,6 +412,26 @@ async function resolveExternalIdentifierFromJwsHeader(
   }
 }
 
+function loadJWK(
+  providedJwk: JWK | undefined,
+  protectedHeader: JwsHeader,
+  jws: JwsJsonGeneral,
+): JWK | undefined {
+  if (providedJwk) {
+    return providedJwk
+  }
+  // TODO? the identityResolver could handle this as well, but it's a really tiny function
```

Comment:
Please create a ticket and link it. Because that really is the sole job of the identity resolution. To have a single place that can resolve any key material, instead of ending up with functions all over the place like we had

Reply:
fixed